docs/4.0/content/images.md

```diff
@@ -69,3 +69,15 @@ Align images with the [helper float classes]({{ site.baseurl }}/docs/{{ site.doc
   <img src="..." class="rounded" alt="...">
 </div>
 {% endhighlight %}
+
+
+## Picture
+
+If you are using the `<picture>` [element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) to specify multiple `<source>` elements for a specific `<img>` make sure to add a `.picture` class to display `.img-*` styles correctly.
```

Comment:
```
`<source>` elements
```
->
```
`<source>` [elements](https://developer.mozilla.org/ru/docs/Web/HTML/Element/source)?
```
?

Reply:
Yeap, you are right.

I'll probably reword the doc and use a utility class instead due to comments on #24176